src/addons/__tests__/update-test.js

```diff
@@ -72,7 +72,10 @@ describe('update', function() {
   });
 
   it('should support apply', function() {
-    expect(update(2, {$apply: function(x) { return x * 2; }})).toEqual(4);
+    function f(x) {
```

Comment:
Does `(x) => x * 2` work here? If so, let's do that – but otherwise this is okay.